css/components.css

```diff
@@ -0,0 +1,12 @@
+@import url("components/alert/alert.css");
+@import url("components/avatar/avatar.css");
+@import url("components/badges/badges.css");
+@import url("components/button/button.css");
+@import url("components/cards/cards.css");
+@import url("components/grid/grid.css");
+@import url("components/image/image.css");
+@import url("components/input/input.css");
+@import url("components/list/list.css");
+@import url("components/navigation/navigation.css");
+@import url("components/slider/slider.css");
+@import url("components/text-utilities/text-utilities.css");
```

Comment:
It is usually not advisable to make changes on the main.css file while working on the slider branch.
Just a small suggestion.
Because our project is small it is fine to do these types of changes.

Reply:
Will keep in mind. Thank you Kevin